app/views/shared/_booking_popup.html.erb

```diff
@@ -0,0 +1,46 @@
+
+<!-- Bootstrap Modal with Booking Form -->
+<div class="modal fade" id="exampleModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body" id="booking-form">
+        <h2 class="modal-title" id="exampleModalLabel">Book this experience now</h2>
+        <% if user_signed_in? %>
+          <div id="offer-info">
```

Comment:
Maybe we should make this into a partial? Same info as offer_details card. Styling and icons will be consistent then

Reply:
Good point, noted in Trello